client/tests/webdriver/pages/createNewTask.page.js

```diff
@@ -0,0 +1,66 @@
+import Page from "./page"
+
+const PAGE_URL = "/tasks/new"
+
+class CreateTask extends Page {
+  get form() {
+    return browser.$("form")
+  }
+
+  get alertSuccess() {
+    return browser.$(".alert-success")
+  }
+
+  get submitButton() {
+    return browser.$("#formBottomSubmit")
+  }
+
+  get shortName() {
+    return browser.$('input[id="shortName"]')
+  }
+
+  get addObjectButton() {
+    return browser.$('button[id="addObjectButton"]')
+  }
+
+  get objectFields() {
+    return browser
+      .$("#custom-fields")
+      .$$("//div[starts-with(@id,'fg-formCustomFields')]")
+  }
+
+  get questionsField() {
+    return browser
+      .$("#custom-fields")
+      .$("//textarea[contains(@id,'questions')]")
+  }
+
+  get questionsFieldWarningText() {
+    return this.questionsField.$("//span[contains(text(), 'Invalid')]")
+  }
+
+  get questionsFieldHelpText() {
+    return this.questionsField.$(".help-block")
```

Comment:
I think this is not the correct selector, it is trying to select the help-block from the current element, and that is a textarea.